http/README.md

```diff
@@ -68,5 +68,84 @@ Table below shows the behaviour of HttpAction depending on provided `responseOpt
 | application/text | false     | JSON           | JSON | _error     | -        |
 | application/text | true      | JSON           | JSON | _error     | -        |
 
+### Node log
+`HttpAction` provides node log, every HTTP call is logged on `INFO` level, failing ones also on `ERROR`. 
+Every HTTP call which have ended with 200 status, but causing exception during processing in `HttpAction` 
+will be treated as succeeding, and the transition will be set to `_error`. Every error is logged.
+
+Node log is presented as `JSON`. 
+
+`JSON` structure of succeeding request on `INFO` level looks like the following (on `ERROR`
+level no logs will be produced):
+```json
+{
+  "request": {
```

Comment:
Can we organise those JSON examples in the form:
```
{
  "request": REQUEST_DATA,
  "response": RESPONSE_DATA,
  "responseBody": RESPONSE_BODY,
  "errors": [ LIST_OF_ERRORS ]
}
```
And then specify what each section contains:

REQUEST_DATA:
```
"path": "/api/endpoint",
"requestHeaders": {
  "Content-Type": "application/json"
}
```
etc.

Now you can specify what is expected in cases:

| Response        | Log level           | Log entries  |
| ------------- |:-------------:| -----:|
| `_success` | INFO | REQUEST_DATA, RESPONSE_DATA, RESPONSE_BODY |
| `_success` | ERROR |  |
| exception occurs and `_error` | INFO      |   REQUEST_DATA, LIST_OF_ERRORS |
| exception occurs and `_error` | ERROR      |   REQUEST_DATA, LIST_OF_ERRORS |
| `_error` (e.g service responds with `500`) | INFO      |   REQUEST_DATA, RESPONSE_DATA, RESPONSE_BODY |
| `_error` (e.g service responds with `500`) | ERROR      |   REQUEST_DATA, RESPONSE_DATA |